homeassistant/components/ambiclimate/climate.py

```diff
@@ -146,32 +146,32 @@ def __init__(self, heater, store):
         """Initialize the thermostat."""
         self._heater = heater
         self._store = store
-        self._attr_unique_id = self._heater.device_id
-        self._attr_name = self._heater.name
+        self._attr_unique_id = heater.device_id
+        self._attr_name = heater.name
         self._attr_device_info = {
             "identifiers": {(DOMAIN, self.unique_id)},
             "name": self.name,
             "manufacturer": "Ambiclimate",
         }
-        self._attr_min_temp = self._heater.get_min_temp()
-        self._attr_max_temp = self._heater.get_max_temp()
+        self._attr_min_temp = heater.get_min_temp()
+        self._attr_max_temp = heater.get_max_temp()
 
-    async def async_set_temperature(self, **kwargs) -> None:
+    async def async_set_temperature(self, **kwargs):
```

Comment:
```suggestion
    async def async_set_temperature(self, **kwargs: Any) -> None:
```

Add `from typing import Any`